lib/core/form/components/widgets/text/text.widget.spec.ts

```diff
@@ -34,12 +36,14 @@ describe('TextWidgetComponent', () => {
         imports: [
             NoopAnimationsModule,
             CoreModule.forRoot()
```

Comment:
Please avoid to use CoreModule, try to import just the modules needed for this component.